packages/stack/src/types.tsx

```diff
@@ -306,6 +315,21 @@ export type StackNavigationOptions = StackHeaderOptions &
      * When `pop` is used, the `pop` animation is applied to the screen being replaced.
      */
     animationTypeForReplace?: 'push' | 'pop';
+    /**
+     * How the screen should animate when pushed or popped.
+     *
+     * Supported values:
+     * - "default": use the platform default animation
+     * - "fade": fade screen in or out
+     * - "fade_from_bottom": fade screen in or out from bottom
+     * - "slide_from_bottom": slide in the new screen from bottom
+     * - "slide_from_right": slide in the new screen from right
+     * - "slide_from_left": slide in the new screen from left
+     * - "none": don't animate the screen
+     *
+     * Only supported on iOS and Android.
```

Comment:
This isn't correct. Animations are supported on all platforms. They are only enabled by default on iOS and Android which can be configured with `animationEnabled: false`